autopush/main.py

```diff
@@ -172,6 +172,10 @@ def _parse_connection(sysargs):
                         help="The maximum size that messages from client " +
                         "can be (e.g. header, data, json formatting, etc.)",
                         default=2048, type=int, env_var="MAX_MESSAGE_SIZE")
+    parser.add_argument('--close_handshake_timeout',
+                        help="The WebSocket closing handshake timeout. Set to "
+                        "0 to disable.", default=0, type=int,
+                        env_var="CLOSE_HANDSHAKE_TIMEOUT")
```

Comment:
According to the docs: http://autobahn.ws/python/reference/autobahn.websocket.html#autobahn.websocket.protocol.WebSocketClientProtocol 1 is the default. I presume that 0 means that no timeout is set and that thing would hang forever?